### ISSUE TITLE
COCO dataset, validation set included?

### ISSUE THREAD

**Issue by bbzhu-jy16** [limuloo/MIGC#12]
Did the validation set of COCO-2014 included in the training stage of your model?

**Comment by LeyRio** [comment]
Hello, thank you for your interest in our work. In that version of the model described in the paper, the training data we used was obtained from the COCO train 2014 dataset, while for evaluation, we used annotations from the COCO val 2014 dataset to construct test samples. Therefore, they are not overlapping.

**Comment by bbzhu-jy16** [comment]
Thanks for your reply, so the open-source model is also trained on the training set only?

**Comment by LeyRio** [comment]
Yes, the open-source model is also trained on the training set only.

**Comment by bbzhu-jy16** [comment]
Thanks for your reply~
